src/api/ledger/parse/ledger.js

```diff
@@ -5,6 +5,11 @@ const removeUndefined = require('./utils').removeUndefined;
 const parseTransaction = require('./transaction');
 import type {GetLedger} from '../types.js';
 
+function parseTransactionWrapper(transaction) {
+  _.assign({}, transaction, {meta: transaction.metaData});
```

Comment:
This has no effect; you have to pass the return value to parseTransaction. Also could you add a unit test to make sure it works?